### PR TITLE
Adding hadronizers and LHE-GEN-SIM config files for AMSB charginos Run3 production

### PR DESCRIPTION
**Pull request by borzari**
Adding hadronizers and LHE-GEN-SIM configuration files for the production of chargino+chargino and chargino+neutralino in the AMSB process using MG5, where the mass difference between chargino and neutralino is small enough to make the chargino decay inside the tracker volume (long-lived particle for EXO disappearing tracks Run 3 analysis). The script uses templates to create hadronizers and LHE-GEN-SIM configuration files for several masses/lifetimes of the chargino and is executed in two steps that are explained in the `README.md` file. Both Run3Summer2022(EE) campaigns are accounted for in the script.

### CODE REVIEW

File: genfragments/ThirteenPointSixTeV/AMSB_chargino/Higgsino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            'JetMatching:setMad = off',
+            'JetMatching:scheme = 1',
+            'JetMatching:merge = on',
+            'JetMatching:jetAlgorithm = 2',
+            'JetMatching:etaJetMax = 5.',
+            'JetMatching:coneRadius = 1.',
+            'JetMatching:slowJetPower = 1',
+            'JetMatching:qCut = %.1f' % (XQCUT * 1.5), #this is the actual merging scale
+            'JetMatching:nQmatch = 5', #4 corresponds to 4-flavour scheme (no matching of b-quarks), 5 for 5-flavour scheme
+            'JetMatching:nJetMax = 1', #number of partons in born matrix element for highest multiplicity
```
Comment (menglu21) [inline genfragments/ThirteenPointSixTeV/AMSB_chargino/Higgsino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py:50]: the process use 4f or 5f model, and what's the jet multiplicity in the process card

File: genfragments/ThirteenPointSixTeV/AMSB_chargino/AMSB_chargino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            'JetMatching:setMad = off',
+            'JetMatching:scheme = 1',
+            'JetMatching:merge = on',
+            'JetMatching:jetAlgorithm = 2',
+            'JetMatching:etaJetMax = 5.',
+            'JetMatching:coneRadius = 1.',
+            'JetMatching:slowJetPower = 1',
+            'JetMatching:qCut = %.1f' % (XQCUT * 1.5), #this is the actual merging scale
+            'JetMatching:nQmatch = 5', #4 corresponds to 4-flavour scheme (no matching of b-quarks), 5 for 5-flavour scheme
+            'JetMatching:nJetMax = 1', #number of partons in born matrix element for highest multiplicity
```
Comment (menglu21) [inline genfragments/ThirteenPointSixTeV/AMSB_chargino/AMSB_chargino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py:44]: the process use 4f or 5f model, and what's the jet multiplicity in the process card

File: genfragments/ThirteenPointSixTeV/AMSB_chargino/Higgsino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py (hunk 1 above)
Comment (borzari) [reply genfragments/ThirteenPointSixTeV/AMSB_chargino/Higgsino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py:50]: Thank you for pointing it out. I realized that this was from a test I was doing, but the actual value should be 4 (as it also is in the [run_card](https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/AMSB_chargino/run_cards/PROCESS_MMASSGeV_ctauCTAUcm_run_card.dat) and as suggested in this [topic](https://cms-talk.web.cern.ch/t/correct-pdf-to-use-for-exo-disappearing-tracks-analysis/14867/2)). Changing it just now

File: genfragments/ThirteenPointSixTeV/AMSB_chargino/AMSB_chargino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py (hunk 2 above)
Comment (borzari) [reply genfragments/ThirteenPointSixTeV/AMSB_chargino/AMSB_chargino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py:44]: Thank you for pointing it out. I realized that this was from a test I was doing, but the actual value should be 4 (as it also is in the [run_card](https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/AMSB_chargino/run_cards/PROCESS_MMASSGeV_ctauCTAUcm_run_card.dat) and as suggested in this [topic](https://cms-talk.web.cern.ch/t/correct-pdf-to-use-for-exo-disappearing-tracks-analysis/14867/2)). Changing it just now

File: genfragments/ThirteenPointSixTeV/AMSB_chargino/Higgsino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py (hunk 1 above)
Comment (borzari) [reply genfragments/ThirteenPointSixTeV/AMSB_chargino/Higgsino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py]: In the [proc_card](https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/AMSB_chargino/process_cards/AMSB_chargino_MMASSGeV_ctauCTAUcm_proc_card.dat) I am using one extra hard jet

File: genfragments/ThirteenPointSixTeV/AMSB_chargino/AMSB_chargino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py (hunk 2 above)
Comment (borzari) [reply genfragments/ThirteenPointSixTeV/AMSB_chargino/AMSB_chargino_M-XXXGeV_CTau-YYYcm_TuneCP5_PSweights_13p6TeV_madgraph5_pythia8_cff.py]: In the [proc_card](https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/AMSB_chargino/process_cards/AMSB_chargino_MMASSGeV_ctauCTAUcm_proc_card.dat) I am using one extra hard jet

**Pull request merged**